src/mod/irc.mod/chan.c

```diff
@@ -1335,6 +1335,61 @@ static int got354(char *from, char *msg)
   return 0;
 }
 
+/* got 353: NAMES
+ * <server> 353 <chan> :[+/@]nick [+/@]nick ....
```

Comment:
where is the "=/@" prefix in the example?

Reply:
modified example